command.c

```diff
@@ -1794,7 +1794,7 @@ bool command_event(enum event_command cmd, void *data)
       case CMD_EVENT_LOAD_CORE:
          command_event(CMD_EVENT_LOAD_CORE_PERSIST, NULL);
 #ifndef HAVE_DYNAMIC
-         command_event(CMD_EVENT_QUIT, NULL);
+         // command_event(CMD_EVENT_QUIT, NULL);
 #endif
```

Comment:
Nope! I fat fingered a delete and I thought I committed a change that undid this but I must not have pushed this. In any case, this PR isn't meant to be merged I guess and I need to come up with a better way to do this just for iOS 10